falcon_challenge/evaluator.py

```diff
@@ -365,20 +385,26 @@ def predict_files(self, decoder: BCIDecoder, eval_files: List):
         # returns triple dict, keyed by datafile hash and contains preds, targets, and eval_mask respective. also returns lists compute_time and neural_time
         # TODO this does not return uniquely identifiable data if eval_files is partial, e.g. if we only has set 2 of a day with 2 sets, we'll happily just provide partial predictions.
         # Pre-loop before starting batch loads
+        
         file_neural_data = []
         file_trial_change = []
         file_targets = []
         file_eval_mask = []
         all_neural_times = []
         datafiles = list(sorted(eval_files))
+
         for datafile in datafiles:
             if not datafile.exists():
                 raise FileNotFoundError(f"File {datafile} not found.")
+        
             neural_data, decoding_targets, trial_change, eval_mask = load_nwb(datafile, dataset=self.dataset)
+            
             file_neural_data.append(neural_data)
             file_trial_change.append(trial_change)
             file_targets.append(decoding_targets)
             file_eval_mask.append(eval_mask)
+
+            if self.dataset == FalconTask.b1: BIN_SIZE = 1/30000
```

Comment:
not ideal, maybe declare as class prop, Set in the initializer

Reply:
Lmk if you find the current solution appropriate, setting BIN_SIZE in the initializer of FalconEvaluator